src/SplitPane.js

```diff
@@ -65,6 +68,9 @@ class SplitPane extends React.Component {
         size,
       },
     };
+
+    this.setPane1 = node => (this.pane1 = node);
```

Comment:
Can we declare these functions in the class definition instead of the constructor?

Reply:
No I don't think you can, because they need access to the `this` pointer and there's no way to get a fixed function reference to a class method that is properly `bind`ed. You see the same pattern in other high-performance React code, for example:

https://github.com/bvaughn/react-virtualized/blob/master/source/List/List.example.js

I hear there's an ES7 proposal to allow arrow functions in the class definition, but at the moment it's experimental. Seems it can be used with Babel though; see [here](https://stackoverflow.com/a/31362350/2659595).

Reply:
Yes you can. We use babel to transform in this project as well. Thanks! Will merge after that change